io.github.igorlogius.scr2ppm.yml

```
@@ -0,0 +1,22 @@
+app-id: io.github.igorlogius.scr2ppm
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: scr2ppm
+finish-args:
+  - --socket=x11
```

```suggestion
  - --socket=x11
  - --share=ipc
```

IPC are needed for X11.

---

Are you sure ipc is necessary?  My local tests all worked fine with just  `--socket=x11`

---

because x11 just take the slow path in that case.

---

You mean the unix socket? 
Well, i can change it if you know that it makes a difference.  
Normally i would like to request the smallest permission set that is necessary, but ... i guess that can still be changed later.